AutoPartsShop.API/Repositories/Interfaces/IProductRepository.cs

```diff
@@ -0,0 +1,13 @@
+﻿using AutoPartsShop.API.Models;
+
+namespace AutoPartsShop.API.Repositories.Interfaces
+{
+    public interface IProductRepository
+    {
+        Task<IEnumerable<Product>> GetItems();
+        Task<IEnumerable<ProductCategory>> GetCategories();
```

Comment:
All methods referring to anything other that products should be moved in a different interface.
So, in this scenario, `GetCategories` and `GetCategory` should be moved into an `ICategoryRepository` (and the methods implemented into a `CategoryRepository` class) in order to have a cleaner code.